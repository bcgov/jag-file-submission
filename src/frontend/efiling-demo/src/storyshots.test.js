import initStoryshots, {
  Stories2SnapsConverter,
} from "@storybook/addon-storyshots";
import { render, waitFor } from "@testing-library/react";
import { createSerializer } from "enzyme-to-json";

const converter = new Stories2SnapsConverter();

const runTest = async (story, context) => {
  const filename = converter.getSnapshotFileName(context);

  if (!filename) {
    return;
  }

  const storyElement = story.render();

  const { asFragment } = render(storyElement);

  await waitFor(() => {});

  expect(asFragment()).toMatchSpecificSnapshot(filename);
};

initStoryshots({
  asyncJest: true,
  snapshotSerializers: [createSerializer()],
  test: ({ story, context, done }) => {
    runTest(story, context).then(done);
  },
});
