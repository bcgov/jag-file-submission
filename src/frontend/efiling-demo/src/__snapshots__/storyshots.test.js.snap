// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Cancel Default 1`] = `
<main>
  <header>
    <nav
      className="container-fluid navbar navbar-expand-lg navbar-dark"
    >
      <div
        aria-labelledby="title"
        className="navbar-brand pointer"
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-none d-md-block pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-logo-rev-en.svg"
        width={181}
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-md-none pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-symbol-rev.svg"
        width={64}
      />
      <div
        aria-labelledby="title"
        className="pointer navbar-brand nav-item nav-link"
      />
      <div
        className="navbar-brand"
        id="title"
      >
        eFiling Demo Client
      </div>
    </nav>
  </header>
  <div
    className="page"
  >
    <div
      className="content col-md-12"
    >
      <p>
        You have cancelled your submission. Please click the button below to go back home.
      </p>
      <div>
        <button
          className="normal-white btn"
          data-test-id="return-home-btn"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Return home
        </button>
      </div>
    </div>
  </div>
  <footer
    className="footer"
  >
    <nav
      className="navbar navbar-expand-sm navbar-dark justify-content-end"
    >
      <button
        className="navbar-toggler"
        data-target="#footerBar"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse flex-grow-0"
        id="footerBar"
      >
        <ul
          className="navbar-nav text-right"
        >
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca"
              rel="noopener noreferrer"
              target="_blank"
            >
              BC Government
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
              rel="noopener noreferrer"
              target="_blank"
            >
              Disclaimer
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/privacy"
              rel="noopener noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
              rel="noopener noreferrer"
              target="_blank"
            >
              Accessibility
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/copyright"
              rel="noopener noreferrer"
              target="_blank"
            >
              Copyright
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </footer>
</main>
`;

exports[`Storyshots Cancel Mobile 1`] = `
<main>
  <header>
    <nav
      className="container-fluid navbar navbar-expand-lg navbar-dark"
    >
      <div
        aria-labelledby="title"
        className="navbar-brand pointer"
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-none d-md-block pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-logo-rev-en.svg"
        width={181}
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-md-none pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-symbol-rev.svg"
        width={64}
      />
      <div
        aria-labelledby="title"
        className="pointer navbar-brand nav-item nav-link"
      />
      <div
        className="navbar-brand"
        id="title"
      >
        eFiling Demo Client
      </div>
    </nav>
  </header>
  <div
    className="page"
  >
    <div
      className="content col-md-12"
    >
      <p>
        You have cancelled your submission. Please click the button below to go back home.
      </p>
      <div>
        <button
          className="normal-white btn"
          data-test-id="return-home-btn"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Return home
        </button>
      </div>
    </div>
  </div>
  <footer
    className="footer"
  >
    <nav
      className="navbar navbar-expand-sm navbar-dark justify-content-end"
    >
      <button
        className="navbar-toggler"
        data-target="#footerBar"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse flex-grow-0"
        id="footerBar"
      >
        <ul
          className="navbar-nav text-right"
        >
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca"
              rel="noopener noreferrer"
              target="_blank"
            >
              BC Government
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
              rel="noopener noreferrer"
              target="_blank"
            >
              Disclaimer
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/privacy"
              rel="noopener noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
              rel="noopener noreferrer"
              target="_blank"
            >
              Accessibility
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/copyright"
              rel="noopener noreferrer"
              target="_blank"
            >
              Copyright
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </footer>
</main>
`;

exports[`Storyshots Error Default 1`] = `
<main>
  <header>
    <nav
      className="container-fluid navbar navbar-expand-lg navbar-dark"
    >
      <div
        aria-labelledby="title"
        className="navbar-brand pointer"
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-none d-md-block pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-logo-rev-en.svg"
        width={181}
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-md-none pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-symbol-rev.svg"
        width={64}
      />
      <div
        aria-labelledby="title"
        className="pointer navbar-brand nav-item nav-link"
      />
      <div
        className="navbar-brand"
        id="title"
      >
        eFiling Demo Client
      </div>
    </nav>
  </header>
  <div
    className="page"
  >
    <div
      className="content col-md-12"
    >
      <p>
        The following error occurred with your submission. Please click the button below to go back home.
      </p>
      <p>
        <b>
          Could not communicate with CSO.
        </b>
      </p>
      <br />
      <div>
        <button
          className="normal-white btn"
          data-test-id=""
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Return home
        </button>
      </div>
    </div>
  </div>
  <footer
    className="footer"
  >
    <nav
      className="navbar navbar-expand-sm navbar-dark justify-content-end"
    >
      <button
        className="navbar-toggler"
        data-target="#footerBar"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse flex-grow-0"
        id="footerBar"
      >
        <ul
          className="navbar-nav text-right"
        >
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca"
              rel="noopener noreferrer"
              target="_blank"
            >
              BC Government
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
              rel="noopener noreferrer"
              target="_blank"
            >
              Disclaimer
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/privacy"
              rel="noopener noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
              rel="noopener noreferrer"
              target="_blank"
            >
              Accessibility
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/copyright"
              rel="noopener noreferrer"
              target="_blank"
            >
              Copyright
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </footer>
</main>
`;

exports[`Storyshots Error Mobile 1`] = `
<main>
  <header>
    <nav
      className="container-fluid navbar navbar-expand-lg navbar-dark"
    >
      <div
        aria-labelledby="title"
        className="navbar-brand pointer"
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-none d-md-block pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-logo-rev-en.svg"
        width={181}
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-md-none pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-symbol-rev.svg"
        width={64}
      />
      <div
        aria-labelledby="title"
        className="pointer navbar-brand nav-item nav-link"
      />
      <div
        className="navbar-brand"
        id="title"
      >
        eFiling Demo Client
      </div>
    </nav>
  </header>
  <div
    className="page"
  >
    <div
      className="content col-md-12"
    >
      <p>
        The following error occurred with your submission. Please click the button below to go back home.
      </p>
      <p>
        <b>
          Could not communicate with CSO.
        </b>
      </p>
      <br />
      <div>
        <button
          className="normal-white btn"
          data-test-id=""
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Return home
        </button>
      </div>
    </div>
  </div>
  <footer
    className="footer"
  >
    <nav
      className="navbar navbar-expand-sm navbar-dark justify-content-end"
    >
      <button
        className="navbar-toggler"
        data-target="#footerBar"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse flex-grow-0"
        id="footerBar"
      >
        <ul
          className="navbar-nav text-right"
        >
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca"
              rel="noopener noreferrer"
              target="_blank"
            >
              BC Government
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
              rel="noopener noreferrer"
              target="_blank"
            >
              Disclaimer
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/privacy"
              rel="noopener noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
              rel="noopener noreferrer"
              target="_blank"
            >
              Accessibility
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/copyright"
              rel="noopener noreferrer"
              target="_blank"
            >
              Copyright
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </footer>
</main>
`;

exports[`Storyshots Home Default 1`] = `
<main>
  <header>
    <nav
      className="container-fluid navbar navbar-expand-lg navbar-dark"
    >
      <div
        aria-labelledby="title"
        className="navbar-brand pointer"
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-none d-md-block pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-logo-rev-en.svg"
        width={181}
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-md-none pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-symbol-rev.svg"
        width={64}
      />
      <div
        aria-labelledby="title"
        className="pointer navbar-brand nav-item nav-link"
      />
      <div
        className="navbar-brand"
        id="title"
      >
        eFiling Demo Client
      </div>
    </nav>
  </header>
  <div
    className="page"
  >
    <div
      className="content col-md-12"
    >
      <div>
        <label
          className="label"
          htmlFor="textInputId"
        >
          <div
            className="input-label"
          >
            Account GUID
          </div>
          <span
            className="mandatory"
            id="asterisk"
          >
            *
          </span>
           
          <span
            className="note"
          />
        </label>
        <input
          className="editable-white"
          id="textInputId"
          onChange={[Function]}
          placeholder="77da92db-0791-491e-8c58-1a969e67d2fa"
          type="text"
        />
        <br />
        <span
          className="error"
        />
      </div>
      <br />
      <div>
        <button
          className="normal-blue btn"
          data-test-id="generate-url-btn"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Generate URL
        </button>
      </div>
      <br />
    </div>
  </div>
  <footer
    className="footer"
  >
    <nav
      className="navbar navbar-expand-sm navbar-dark justify-content-end"
    >
      <button
        className="navbar-toggler"
        data-target="#footerBar"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse flex-grow-0"
        id="footerBar"
      >
        <ul
          className="navbar-nav text-right"
        >
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca"
              rel="noopener noreferrer"
              target="_blank"
            >
              BC Government
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
              rel="noopener noreferrer"
              target="_blank"
            >
              Disclaimer
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/privacy"
              rel="noopener noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
              rel="noopener noreferrer"
              target="_blank"
            >
              Accessibility
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/copyright"
              rel="noopener noreferrer"
              target="_blank"
            >
              Copyright
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </footer>
</main>
`;

exports[`Storyshots Home Mobile 1`] = `
<main>
  <header>
    <nav
      className="container-fluid navbar navbar-expand-lg navbar-dark"
    >
      <div
        aria-labelledby="title"
        className="navbar-brand pointer"
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-none d-md-block pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-logo-rev-en.svg"
        width={181}
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-md-none pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-symbol-rev.svg"
        width={64}
      />
      <div
        aria-labelledby="title"
        className="pointer navbar-brand nav-item nav-link"
      />
      <div
        className="navbar-brand"
        id="title"
      >
        eFiling Demo Client
      </div>
    </nav>
  </header>
  <div
    className="page"
  >
    <div
      className="content col-md-12"
    >
      <div>
        <label
          className="label"
          htmlFor="textInputId"
        >
          <div
            className="input-label"
          >
            Account GUID
          </div>
          <span
            className="mandatory"
            id="asterisk"
          >
            *
          </span>
           
          <span
            className="note"
          />
        </label>
        <input
          className="editable-white"
          id="textInputId"
          onChange={[Function]}
          placeholder="77da92db-0791-491e-8c58-1a969e67d2fa"
          type="text"
        />
        <br />
        <span
          className="error"
        />
      </div>
      <br />
      <div>
        <button
          className="normal-blue btn"
          data-test-id="generate-url-btn"
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Generate URL
        </button>
      </div>
      <br />
    </div>
  </div>
  <footer
    className="footer"
  >
    <nav
      className="navbar navbar-expand-sm navbar-dark justify-content-end"
    >
      <button
        className="navbar-toggler"
        data-target="#footerBar"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse flex-grow-0"
        id="footerBar"
      >
        <ul
          className="navbar-nav text-right"
        >
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca"
              rel="noopener noreferrer"
              target="_blank"
            >
              BC Government
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
              rel="noopener noreferrer"
              target="_blank"
            >
              Disclaimer
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/privacy"
              rel="noopener noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
              rel="noopener noreferrer"
              target="_blank"
            >
              Accessibility
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/copyright"
              rel="noopener noreferrer"
              target="_blank"
            >
              Copyright
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </footer>
</main>
`;

exports[`Storyshots Success Default 1`] = `
<main>
  <header>
    <nav
      className="container-fluid navbar navbar-expand-lg navbar-dark"
    >
      <div
        aria-labelledby="title"
        className="navbar-brand pointer"
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-none d-md-block pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-logo-rev-en.svg"
        width={181}
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-md-none pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-symbol-rev.svg"
        width={64}
      />
      <div
        aria-labelledby="title"
        className="pointer navbar-brand nav-item nav-link"
      />
      <div
        className="navbar-brand"
        id="title"
      >
        eFiling Demo Client
      </div>
    </nav>
  </header>
  <div
    className="page"
  >
    <div
      className="content col-md-12"
    >
      <p>
        You have successfully completed your submission. Please click the button below to go back home.
      </p>
      <div>
        <button
          className="normal-white btn"
          data-test-id=""
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Return home
        </button>
      </div>
    </div>
  </div>
  <footer
    className="footer"
  >
    <nav
      className="navbar navbar-expand-sm navbar-dark justify-content-end"
    >
      <button
        className="navbar-toggler"
        data-target="#footerBar"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse flex-grow-0"
        id="footerBar"
      >
        <ul
          className="navbar-nav text-right"
        >
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca"
              rel="noopener noreferrer"
              target="_blank"
            >
              BC Government
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
              rel="noopener noreferrer"
              target="_blank"
            >
              Disclaimer
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/privacy"
              rel="noopener noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
              rel="noopener noreferrer"
              target="_blank"
            >
              Accessibility
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/copyright"
              rel="noopener noreferrer"
              target="_blank"
            >
              Copyright
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </footer>
</main>
`;

exports[`Storyshots Success Mobile 1`] = `
<main>
  <header>
    <nav
      className="container-fluid navbar navbar-expand-lg navbar-dark"
    >
      <div
        aria-labelledby="title"
        className="navbar-brand pointer"
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-none d-md-block pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-logo-rev-en.svg"
        width={181}
      />
      <img
        alt="B.C. Government Logo"
        className="img-fluid d-md-none pointer"
        height="44"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="presentation"
        src="undefined/images/bcid-symbol-rev.svg"
        width={64}
      />
      <div
        aria-labelledby="title"
        className="pointer navbar-brand nav-item nav-link"
      />
      <div
        className="navbar-brand"
        id="title"
      >
        eFiling Demo Client
      </div>
    </nav>
  </header>
  <div
    className="page"
  >
    <div
      className="content col-md-12"
    >
      <p>
        You have successfully completed your submission. Please click the button below to go back home.
      </p>
      <div>
        <button
          className="normal-white btn"
          data-test-id=""
          disabled={false}
          onClick={[Function]}
          type="button"
        >
          Return home
        </button>
      </div>
    </div>
  </div>
  <footer
    className="footer"
  >
    <nav
      className="navbar navbar-expand-sm navbar-dark justify-content-end"
    >
      <button
        className="navbar-toggler"
        data-target="#footerBar"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse flex-grow-0"
        id="footerBar"
      >
        <ul
          className="navbar-nav text-right"
        >
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca"
              rel="noopener noreferrer"
              target="_blank"
            >
              BC Government
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/disclaimer"
              rel="noopener noreferrer"
              target="_blank"
            >
              Disclaimer
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/privacy"
              rel="noopener noreferrer"
              target="_blank"
            >
              Privacy
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/accessibility"
              rel="noopener noreferrer"
              target="_blank"
            >
              Accessibility
            </a>
          </li>
          <li
            className="nav-item m-1"
          >
            <a
              className="nav-link"
              href="https://www2.gov.bc.ca/gov/content/home/copyright"
              rel="noopener noreferrer"
              target="_blank"
            >
              Copyright
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </footer>
</main>
`;
